### PR TITLE
[ie/IPrima] Fix extractor

### LINKED ISSUE #6100: Add support for zoom.iprima.cz
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a new site support request
- [X] I've verified that I'm running yt-dlp version **2023.01.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that none of provided URLs [violate any copyrights](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy) or contain any [DRM](https://en.wikipedia.org/wiki/Digital_rights_management) to the best of my knowledge
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and am willing to share it if required

### Region

Czech Republic / Czechia

### Example URLs

-Single video: https://zoom.iprima.cz/porady/valecni-kseftari-druha-svetova-z-druhe-ruky/blitz
-Playlist: https://zoom.iprima.cz/porady/valecni-kseftari-druha-svetova-z-druhe-ruky

### Provide a description that is worded well enough to be understood

I have no problems downloading from prima.iprima.cz website (for example: https://prima.iprima.cz/zoo-0/sestru-nezradim) however if I try to download from the zoom.iprima.cz using the same credentials I recieve an error (see below).

I am able to successfully log in to both of these sites (they use the same account) and watch the videos mentioned without any problems directly from the website. 

The credentials I've used:
Username: bejote2396@breazeim.com
Password: qwerty

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', '--username', 'PRIVATE', '--password', 'PRIVATE', 'https://zoom.iprima.cz/porady/valecni-kseftari-druha-svetova-z-druhe-ruky/blitz']
[debug] Encodings: locale cp1250, fs utf-8, pref cp1250, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2023.01.06 [6becd25] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.19045-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: none
[debug] Optional libraries: Cryptodome-3.16.0, brotli-1.0.9, certifi-2022.12.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1760 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2023.01.06, Current version: 2023.01.06
yt-dlp is up to date (2023.01.06)
[IPrima] Downloading login page
[IPrima] Logging in
[IPrima] Downloading token
[IPrima] Extracting URL: https://zoom.iprima.cz/porady/valecni-kseftari-druha-svetova-z-druhe-ruky/blitz
[IPrima] blitz: Downloading webpage
ERROR: [IPrima] blitz: Unable to extract real id; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "yt_dlp\extractor\common.py", line 680, in extract
  File "yt_dlp\extractor\iprima.py", line 122, in _real_extract
  File "yt_dlp\extractor\common.py", line 1228, in _search_regex
```

### PR DESCRIPTION
**Pull request by otovalek**
**IMPORTANT**: PRs without the template will be CLOSED

The Zoom IPrima video ID is found at a different location in the page source ( e.g. at https://zoom.iprima.cz/porady/krasy-kanarskych-ostrovu/tenerife-v-risi-ohne).

Fixes #6100

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [X] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [X] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check all of the following options that apply:
- [X] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [X] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

</details>

### CODE REVIEW

File: yt_dlp/extractor/iprima.py [hunk 1]
```diff
@@ -131,6 +131,7 @@ def _real_extract(self, url):
         video_id = self._search_regex((
             r'productId\s*=\s*([\'"])(?P<id>p\d+)\1',
             r'pproduct_id\s*=\s*([\'"])(?P<id>p\d+)\1',
+            r'let videos\s*=\s*([\'"])(?P<id>p\d+)\1',
```
Comment (bashonly) [inline yt_dlp/extractor/iprima.py:134]:
```suggestion
            r'let\s+videos\s*=\s*([\'"])(?P<id>p\d+)\1',
```

**Review by bashonly — CHANGES_REQUESTED** [review]
do the current test cases require this new pattern? if not, please add a test that does

File: yt_dlp/extractor/iprima.py (hunk 1 above)
Comment (otovalek) [reply yt_dlp/extractor/iprima.py]: In 2be0239ee18c9b3414e3d232035338bc5a8f3b69, the a testcase for zoom.iprima.cz added.

File: yt_dlp/extractor/iprima.py [hunk 2]
```diff
@@ -176,7 +197,7 @@ def _real_extract(self, url):
         final_result = self._search_json_ld(webpage, video_id, default={})
         final_result.update({
             'id': video_id,
-            'title': title,
+            'title': final_result.get('title', title),
```
Comment (bashonly) [inline yt_dlp/extractor/iprima.py:200]:
```suggestion
            'title': final_result.get('title') or title,
```

**Review by bashonly — APPROVED** [review]

**Comment by bashonly** [comment]
LGTM. In the future please do not force push

File: yt_dlp/extractor/iprima.py (hunk 2 above)
Comment (otovalek) [reply yt_dlp/extractor/iprima.py:200]: [ie/IPrima] Using "or" instead of get() default.

**Pull request merged**